src/static/menu-mac.json

```diff
@@ -331,6 +331,9 @@
                             "command": true
                         }
                     }
+                },
+                {
+                    "id": "ADD_VECTOR_MASK"
```

Comment:
No shortcut for this one?

On that note, can we have "Edit Vector Mask" be one menu item, and it adds it if there isn't one?